rfcs/20221124-monorepo.md

```diff
@@ -79,13 +75,17 @@ First we will merge the API and the frontend repos into `WordPress/openverse`. T
 
 1. The merge of two JavaScript codebases provides fertile ground for testing `pnpm` workspaces.
 
+   - It also allows us to merge the browser extension later and split the design system/component library stuff into a separate package.
+
 1. The API is already organised by stack folders so the `frontend/` directory will fit right in with the others like `api/` and `ingestion_server/`. Similarly the scripts repo is nicely organised in folders, reducing conflicts.
 
 1. The API and frontend share identical tooling for Git hooks, linting and formatting. We will fight our tools less and encounter minimal friction.
 
-   - The frontend's approach for `pre-commit` inspired the RFC for expaning this type of usage to the API as well!
+   - The frontend's approach for `pre-commit` expanded this type of usage to the API as well!
+
+   - We're expanding the use of double-quoted strings to JavaScript to further unify our style guides.
 
-1. The entire system can be integration tested during releases. The real API, populated with test data, can even replace the Talkback server as long as we can turn off all network calls and enable 100% reliable output.
+1. The entire system can be integration tested during releases. The real API, populated with test data, can replace the Talkback server as long as we disable network calls and make output deterministic.
```

Comment:
I love this! 🚀

Reply:
We would have to make sure that the API response types _never_ differ from the sample data, though. For example, if we say that the license URL is always present, and it's present in the test data, but absent in the real API responses, that would break the app even though the tests pass.

Reply:
Next step would be integrating the catalog so the supplier of the data would also be a part of the tests 👍.